### PR TITLE
docs(custom-certs): add Java configuration section

### PR DESCRIPTION
**Pull request by viceice**
- #3

### CODE REVIEW

File: docs/custom-root-ca.md [hunk 1]
```diff
@@ -31,3 +43,16 @@ docker run --rm -it \
   -e NODE_EXTRA_CA_CERTS=/my-root-ca.crt \
   containerbase/buildpack bash
 ```
+
+### Java
+
+For java you need to mount your own certificate store to `/opt/buildpack/ssl/cacerts`.
```
Comment (viceice) [inline docs/custom-root-ca.md:49]:
```suggestion
For Java you need to mount your own certificate store to `/opt/buildpack/ssl/cacerts`.
```

File: docs/custom-root-ca.md [hunk 2]
```diff
@@ -31,3 +43,16 @@ docker run --rm -it \
   -e NODE_EXTRA_CA_CERTS=/my-root-ca.crt \
   containerbase/buildpack bash
 ```
+
+### Java
+
+For Java you need to mount your own certificate store to `/opt/buildpack/ssl/cacerts`.
```
Comment (rarkins) [inline docs/custom-root-ca.md:49]: How is this URL used when it's only mapped at runtime? e.g. will any java we install automatically look for this URL even if it wasn't present at the time of prepare-tool?

Comment (viceice) [reply docs/custom-root-ca.md:49]: - `/opt/buildpack/ssl` is created by `install-buildpack`
- `/opt/buildpack/ssl/cacerts` is created by `prepare-tool java` if not exists
- when running `install-tool java <version>` as root, it automatically runs `prepare-tool java`

Comment (rarkins) [reply docs/custom-root-ca.md:49]: So the certs file always exists and is expected by Java to be present? That way when it's overridden at runtime Java is already configure to use it?

Comment (viceice) [reply docs/custom-root-ca.md:49]: yes, it's symlinked to the normal path
https://github.com/containerbase/buildpack/blob/5aed1e46fb27b0b03a1aaa4fd583546d24e11e4c/src/usr/local/buildpack/utils/java.sh#L74-L78

**Review by rarkins — APPROVED** [review]

**Pull request merged**